### PR TITLE
[webui] fix borked /monitor/old which got broken with HAML conversion

### PR DESCRIPTION
**Pull request by mmohring**
This fixes the broken _building_table.html.haml .

### CODE REVIEW

File: src/api/app/views/webui/monitor/_building_table.html.haml [hunk 1]
```diff
@@ -53,6 +53,3 @@
         'bPaginate': false,
         'oSearch': {'sSearch': '#{@project_filter}', 'bRegex': false, 'bSmart': false},
       });
-- else
```
Comment (mdeniz) [inline src/api/app/views/webui/monitor/_building_table.html.haml]: Why removing the else part?

**Comment by mdeniz** [comment]
Thanks @mmohring. Beside the remove of the `else` part it LGTM.

File: src/api/app/views/webui/monitor/_building_table.html.haml (hunk 1 above)
Comment (mmohring) [inline src/api/app/views/webui/monitor/_building_table.html.haml]: I think the else case will never be called. Somewhere else the output "All xxx hosts are idle!" will be generated and put into the page.

Comment (mdeniz) [inline src/api/app/views/webui/monitor/_building_table.html.haml]: Yes true. Thanks Martin

**Pull request merged**